downstream/assemblies/hub/assembly-collections-and-content-signing-in-pah.adoc

```diff
@@ -0,0 +1,12 @@
+[id="assembly-collections-and-content-signing-in-pah"]
+
+= Collections and content signing in {PrivateHubName}
+
+As an automation creator, you can configure your private automation hub to successfully sign and publish your own content collections while still maintaining access to Red Hat Certified Collections available through your Ansible Automation Platform subscription or community collections available through Ansible Galaxy. In addition, you can configure Ansible-Galaxy CLI to verify collections to ensure that your collections are approved by your organization and have not been changed after it was uploaded to your automation hub.
+
+
+include::modules/proc-configure-content-signing-on-pah.adoc[leveloffset=+2]
+
+include::modules/proc-using-content-signing-services-in-pah.adoc[leveloffset=+2]
+
+include::modules/proc-configure-ansible-galaxy-cli-verify.adoc[leveloffset=2]
```

Comment:
```suggestion
include::automation-hub/proc-configure-content-signing-on-pah.adoc[leveloffset=+2]

include::automation-hub/proc-using-content-signing-services-in-pah.adoc[leveloffset=+2]

include::automation-hub/proc-configure-ansible-galaxy-cli-verify.adoc[leveloffset=2]
```

Reply:
Thanks, @ariordan-redhat! I figured I had these pointing to the wrong directory and had updated them before I viewed your comment here. Copying and pasting got me in trouble. ; - )